Dependencies/ShaderFixes/uscv.hlsl

```diff
@@ -0,0 +1,28 @@
+void main(
```

Comment:
Is there any way to integrate this directly into the SBS shader as a subroutine?

It would be very nice if we could contain all the variants in the single VS/PS pair.